logline.go

```diff
@@ -183,11 +198,12 @@ func (l *LogLine) ToText(buf []byte) []byte {
 		buf = newbuf
 	}
 
-	done := false
-	for fc := FieldIndex(0); fc < LogLineNumFields && !done; fc++ {
+	for fc := FieldIndex(0); fc < LogLineNumFields; fc++ {
 		buf = append(buf, l.Get(fc)...)
+		if fc >= FieldIndex(lastw) && fc >= FieldIndex(lastr) {
```

Comment:
if we compute the minimum between `lastw` and `lastr` before the `for` statement, then we can perform a single comparison here, instead of 2

Reply:
good idea!